CustomItems/Items/Scp2818.cs

```diff
@@ -0,0 +1,66 @@
+namespace CustomItems.Items
+{
+    using System.Collections.Generic;
+    using Exiled.API.Features;
+    using Exiled.CustomItems.API;
+    using Exiled.CustomItems.API.Features;
+    using Exiled.CustomItems.API.Spawn;
+    using Exiled.Events.EventArgs;
+    using YamlDotNet.Serialization;
+
+    /// <summary>
+    /// A gun that kills you.
+    /// </summary>
+    public class Scp2818 : CustomWeapon
+    {
+        /// <inheritdoc/>
+        public override uint Id { get; set; } = 14;
+
+        /// <inheritdoc/>
+        public override string Name { get; set; } = "SCP-2818";
+
+        /// <inheritdoc/>
+        public override string Description { get; set; } =
+            "When this weapon is fired, it uses the biomass of the shooter as the bullet.";
+
+        /// <inheritdoc/>
+        public override SpawnProperties SpawnProperties { get; set; } = new SpawnProperties
+        {
+            Limit = 1,
```

Comment:
Shouldn't this be set at 2?

Reply:
Why?

Reply:
Because the item inside the HCZ harmory would never have a chance to spawn.

Reply:
Corrected the spawn chances so that both may have a chance to spawn, but neither are guaranteed.